worker/runc/runc_test.go

```diff
@@ -127,7 +134,8 @@ func TestRuncWorker(t *testing.T) {
 	target, err = lm.Mount()
 	require.NoError(t, err)
 
-	dt, err := ioutil.ReadFile(filepath.Join(target, "bar"))
```

Comment:
Please add a link to the issue as a code comment

Reply:
Comment is also added.